include/fastgltf/parser.hpp

```diff
@@ -268,13 +273,21 @@ namespace fastgltf {
         constexpr std::string_view KHR_texture_basisu = "KHR_texture_basisu";
         constexpr std::string_view KHR_texture_transform = "KHR_texture_transform";
         constexpr std::string_view MSFT_texture_dds = "MSFT_texture_dds";
+#if FASTGLTF_ENABLE_DEPRECATED_EXT
+        constexpr std::string_view KHR_materials_pbrSpecularGlossiness = "KHR_materials_pbrSpecularGlossiness";
+#endif
```

Comment:
I haven't respected the alphabetical order here, to eventually group the deprecated extensions in the same area.
Tell me if you want me to change that.

Reply:
yeah seems fine. Though if you wanna do it like that a empty line  between them would make sense imo.

Reply:
Like it is now ?

Reply:
yup